datatypes/src/commonMain/kotlin/at/asitplus/crypto/datatypes/CryptoSignature.kt

```diff
@@ -71,11 +58,12 @@ sealed class CryptoSignature(
         override val descriptor: SerialDescriptor
             get() = PrimitiveSerialDescriptor("CryptoSignature", PrimitiveKind.STRING)
 
-        override fun deserialize(decoder: Decoder): RSAorHMAC =
-            RSAorHMAC(decoder.decodeString().encodeToByteArray())
+        override fun deserialize(decoder: Decoder): CryptoSignature {
+            return CryptoSignature.decodeFromDer(decoder.decodeString().decodeToByteArray(Base64UrlStrict))
+        }
 
         override fun serialize(encoder: Encoder, value: CryptoSignature) {
-            encoder.encodeString(value.serialize())
+            encoder.encodeString(value.encodeToDer().encodeToString(Base64UrlStrict))
```

Comment:
Why is Base64-URL the default for serializing `CryptoSignature`?

Reply:
i'll change that to regualr base64. then again. the serialisation is for debugging purposes only and this needs to be documented